IdentityCore/tests/automation/ui_tests_lib/XCUIElement+CrossPlat.m

```diff
@@ -39,8 +39,8 @@ - (void)msidPasteText:(NSString *)text application:(__unused XCUIApplication *)a
 #if TARGET_OS_IPHONE
     [self typeText:text];
 #else
-    [[NSPasteboard generalPasteboard] declareTypes:[NSArray arrayWithObject:NSStringPboardType] owner:nil];
-    [[NSPasteboard generalPasteboard] setString:text forType:NSStringPboardType];
+    [[NSPasteboard generalPasteboard] declareTypes:[NSArray arrayWithObject:NSPasteboardTypeString] owner:nil];
```

Comment:
In macOS 10.15 the NSStringPboardType enum value has been replaced with NSPasteboardTypeString